label_sync/main.go

```diff
@@ -931,6 +931,7 @@ func writeDocs(template string, output string, config Configuration) error {
 func linkify(text string) string {
 	// swap space with dash
 	link := strings.Replace(text, " ", "-", -1)
+	link = strings.ReplaceAll(link, ".", "")
 	// discard some special characters
 	discard, _ := regexp.Compile("[,/]")
```

Comment:
@pacoxu 

sorry I missed this earlier – I think the right place to discard dots would be here:

```suggestion
        discard, _ := regexp.Compile("[.,/]")
```

Reply:
created: https://github.com/kubernetes/test-infra/pull/33465